bridges/AppleAppStoreBridge.php

```diff
@@ -78,94 +94,157 @@ public function getName()
         return parent::getName();
     }
 
-    /**
-     * In case of some platforms, the data is present in the initial response
-     */
-    private function getDataFromShoebox($id, $platform, $country)
+    private function debugLog($message)
     {
-        $uri = $this->makeHtmlUrl($id, $country);
-        $html = getSimpleHTMLDOMCached($uri, 3600);
-        $script = $html->find('script[id="shoebox-ember-data-store"]', 0);
-
-        $json = json_decode($script->innertext, true);
-        return $json['data'];
+        if ($this->getInput('debug')) {
+            error_log("[AppleAppStoreBridge] $message");
+        }
     }
 
-    private function getJWTToken($id, $platform, $country)
+    private function getHtml()
     {
-        $uri = $this->makeHtmlUrl($id, $country);
-
-        $html = getSimpleHTMLDOMCached($uri, 3600);
+        $url = $this->makeHtmlUrl();
+        $this->debugLog("Fetching HTML from: $url");
+        
+        $html = getSimpleHTMLDOM($url);
+        
+        if (!$html) {
```

Comment:
this will never happen because return value is always `\simple_html_dom`, or it throws exception